src/app/code/community/Omise/Gateway/etc/config.xml

```diff
@@ -180,6 +180,13 @@
                 <title>Internet banking</title>
             </omise_offsite_internet_banking>
 
+            <omise_offsite_citipoints>
+                <active>0</active>
+                <payment_action>order</payment_action>
+                <model>omise_gateway/payment_offsitecitipoints</model>
+                <title>Pay with Citipoints</title>
```

Comment:
Is "Citipoints" the correct name? Googling it gives 0 results

Reply:
https://www.citibank.co.th/en/credit-cards/credit-card-rewards.htm

Reply:
City Pay with Points

Reply:
@robinclart @jonrandy Updated